packages/sdk/contractkit/src/kit.ts

```diff
@@ -215,40 +203,22 @@ export class ContractKit {
 
   async getEpochSize(): Promise<number> {
     const validators = await this.contracts.getValidators()
-    const epochSize = await validators.getEpochSize()
-
-    return epochSize.toNumber()
+    return validators.getEpochSizeNumber()
   }
 
   async getFirstBlockNumberForEpoch(epochNumber: number): Promise<number> {
```

Comment:
implementation moved to ValidatorsWrapper

Reply:
I decided that BlockchainParameters was indeed a better spot for this stuff and moved these group of functions over there plus added a test.